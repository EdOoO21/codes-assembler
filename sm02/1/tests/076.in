-52 945 -203 -591 860 -392 -892 -144 724 -514 -424 -383 541 721 -103 2 -527 -830 -315 735 -382 -535 -319 616 90 -877 -985 349 -155 208 842 242 608 -213 521 -516 -321 110 -65 -950 -870 358 288 878 -176 -689 -796
