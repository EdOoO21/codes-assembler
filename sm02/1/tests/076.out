-796
-689
-176
878
288
358
-870
-950
-65
110
-321
-516
521
-213
608
242
842
208
-155
349
-985
-877
90
616
-319
-535
-382
735
-315
-830
-527
2
-103
721
541
-383
-424
-514
724
-144
-892
-392
860
-591
-203
945
-52
